### ISSUE TITLE
Named Query Service

### ISSUE THREAD

**Issue by tomcrane** [dlcs/protagonist#28]
Non-PDF projections of configured queries.
.NET Core Service

Previously part of Orchestrator

**Comment by tomcrane** [comment]
Any more to do on this?

**Comment by donaldgray** [comment]
> Any more to do on this?

Yes - we have NQ support in Orchestrator but this is to split all of that out of Orchestrator and into it's own service.
